test/functions_topic_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from src.functions_topic import *
+
+
+@pytest.mark.parametrize("string1, string2, expected",
+                         [("", "", ""),
+                          (" ", " ", "  "),
+                          ("a", "b", "ab"),
+                          ("Hello", "World", "HelloWorld")])
+def test_concatenate_strings_returns_concatenated_string(string1, string2, expected):
+    assert concatenate_strings(string1, string2) == expected
```

Comment:
Keep it simple. `parametrize` makes sense for actual python projects to reduce bloat, but it's really only useful if you know what you're doing in the first place. For learning, I'd much rather see completely braindead tests.
```python
def test_concat_a_b():
    assert concat("a", "b") == "ab"

def test_concat_empty():
    assert concat("", "") == ""
````

Reply:
In case it's not clear, the idea is
1. The hacked test function I made is for them learning the idea with unit testing but with something that would work on Colab.
2. The pytest is just for the repo

That said, it's a good point about the tests being simple AF anyways for learning purposes _if_ they go digging on the repo.

Reply:
> The pytest is just for the repo

bruh onboard black, your python code is ugly as sin

Reply:
Wow, rude. 

But yeah. I'll get that up and running with a PR doing `setuptools` et al.